src/Assets/Asset.php

```diff
@@ -78,28 +78,28 @@ class Asset {
 	protected array $groups = [];
 
 	/**
-	 * Should the asset be loaded in the footer?
+	 * Whether the asset be loaded in the footer.
```

Comment:
In the docs updates you missed a `should`.

Whether the asset **should** ... etc...

applicable to multiple places